app/src/main/java/com/github/geohunt/app/ui/components/navigation/NavigationController.kt

```diff
@@ -74,6 +72,9 @@ fun NavigationController(
 
     NavHost(navController, startDestination = Route.Home.route, modifier = modifier) {
         composable(Route.Home.route) {
+            Button(onClick = { navController.navigate("challenge-view/95a5a7d8-NULhJTXctE8qZqzJ2l8") }) {
+                Text(text = "Not hardcoded")
```

Comment:
This should be definitely removed hahaha

Reply:
Oups